pmd-ui/src/main/java/net/sourceforge/pmd/util/fxdesigner/util/controls/NodeEditionCodeArea.java

```diff
@@ -31,10 +30,10 @@
 import net.sourceforge.pmd.util.fxdesigner.util.DesignerUtil;
 import net.sourceforge.pmd.util.fxdesigner.util.codearea.AvailableSyntaxHighlighters;
 import net.sourceforge.pmd.util.fxdesigner.util.codearea.HighlightLayerCodeArea;
-import net.sourceforge.pmd.util.fxdesigner.util.codearea.HighlightLayerCodeArea.LayerId;
```

Comment:
For some reason, the eclipse compiler is requiring this import... 
Otherwise down at [line 195](https://github.com/pmd/pmd/pull/1693/files#diff-7c803a44256387e2620e2efbe11bad8eR195) it cannot resolve LayerId...

Reply:
I didn't find a way to suppress an "unused import" warning on IntelliJ so I qualified the reference line 195 instead. Let me know if that works

Reply:
Works with eclipse - I guess, it's a eclipse compiler bug, since it works with openjdk and intellij...